lib/rules/no-current-route-name.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const ERROR_MESSAGE = 'dfs';
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'disallow usage of the `currentRouteName()` test helper',
+      category: 'Testing',
+      recommended: false,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-current-route-name.md',
+    },
+    fixable: null,
+    schema: [],
+  },
+
+  ERROR_MESSAGE,
+
+  create(context) {
+    const aliases = [];
```

Comment:
Generally better to call this something like `currentRouteNameImportAliases` in case we needed to track more imports in the future...

Reply:
I've renamed it to `importAliases` to be a bit more explicit. Since this is inside the `no-current-route-name.js` file I think the `currentRouteName` prefix is a bit redundant. I'm not sure what other imports we would need to track in the future tbh.